iOS/JWPlayerPlugin/JWPlayerViewController.m

```diff
@@ -87,6 +98,7 @@ - (UIButton *)closeButton {
     button.titleLabel.font = [UIFont systemFontOfSize:25];
     [button setTitle:@"X" forState:UIControlStateNormal];
     [button setTitleColor:UIColor.whiteColor forState:UIControlStateNormal];
+    [button setAccessibilityIdentifier:kJWPlayerCloseButton];
```

Comment:
Views\controls may not be accessible via Xcode Accessibility Inspector or other related software if view\control never passed this method: `[view setIsAccessibilityElement:YES];`

Reply:
According to Apple's documentation in regards to this method: 

> The default value for this property is NO unless the receiver is a standard UIKit control, in which case the value is YES.

In this case we are dealing with standard UIKit controls (UIButtons and UIViews), so this button should be accessible via the Accessibility Inspector.